### DISCUSSION TITLE
Graphql "Server cannot be reached"

### DISCUSSION THREAD

**Discussion by AbduRahmanS** [saleor/saleor#10374] [Deployment]
Hey!
I deployed saleor core using docker and dashboard in static on an ec2 instance. For core, I downloaded the 3.6.2 release and upload it to my server and just build the image.
I ran the image with the following Env variables : 
DATABASE_URL (my rds instance endpoint), SECRET_KEY, ALLOWED_CLIENT_HOSTS=localhost,127.0.0.1,https://mydomain.com, ALLOWED_HOSTS=localhost,127.0.0.1,https://mydomain.com
RSA_PRIVATE_KEY
DEBUG=False 
Migrate db, create a superuser and populatedb
Dashboard on mydomain.com/dashboard/, graphql endpoint on mydomain.com/graphql/. All are reachable perfectly but in graphql playground, I'm seeing this error "Server cannot be reached". And in the dashboard, I'm not able to manage products.

What am I doing wrong? I'm stuck on this for hours.
Could someone please help me?

**Comment by karolkielecki** [comment]
Hi @AbduRahmanS,

Could you please check your dev tools, where your playground is trying to connect to, and if there are any errors on the browser level (like cross-origin blocking, etc.)?

**Comment by mercy-thuyle** [comment]
> Hey! I deployed saleor core using docker and dashboard in static on an ec2 instance. For core, I downloaded the 3.6.2 release and upload it to my server and just build the image. I ran the image with the following Env variables : DATABASE_URL (my rds instance endpoint), SECRET_KEY, ALLOWED_CLIENT_HOSTS=localhost,127.0.0.1,https://mydomain.com, ALLOWED_HOSTS=localhost,127.0.0.1,https://mydomain.com RSA_PRIVATE_KEY DEBUG=False Migrate db, create a superuser and populatedb Dashboard on mydomain.com/dashboard/, graphql endpoint on mydomain.com/graphql/. All are reachable perfectly but in graphql playground, I'm seeing this error "Server cannot be reached". And in the dashboard, I'm not able to manage products.
> 
> What am I doing wrong? I'm stuck on this for hours. Could someone please help me?

Hello @AbduRahmanS , did you fixed this? I have the same problem like you.
Hi @karolkielecki , add header in nginx config right?
Example:

add_header Access-Control-Allow-Origin *; 
add_header Access-Control-Allow-Methods 'GET, HEAD'; 
add_header Content-Security-Policy "default-src 'self';"; 
add_header Access-Control-Allow-Origin https://saleor-admin-my-server.vn;
